install/installer/pkg/config/v1/config.go

```diff
@@ -218,11 +217,8 @@ const (
 
 type Resources struct {
 	// todo(sje): add custom validation to corev1.ResourceList
-	Requests      corev1.ResourceList `json:"requests" validate:"required"`
-	Limits        corev1.ResourceList `json:"limits,omitempty"`
-	DynamicLimits *struct {
-		CPU []cpulimit.Bucket // todo(sje): add custom validation
-	} `json:"dynamicLimits,omitempty"`
```

Comment:
Note: this is backwards incompatible change. We load the config "strictly", i.e. removing this field might break existing configuration. We don't want that.

Even though this field hasn't been used (had no effect), removing it now might break someone's config.

IMHO we should do something like this:
```
DeprecatedDyanmicLimits json.RawMessage `json:"dynamicLimits,omitempty" validate:"deprecated"``
```

and add a config validation function which returns `true` but emits a log message that the field is deprecated and has no effect.